src/util/util_luid.cpp

```diff
@@ -18,9 +18,12 @@ namespace dxvk {
     while (s_luids.size() < newLuidCount) {
       LUID luid = { 0, 0 };
 
+#ifdef _WIN32
```

Comment:
Cleaner approach might be to add an implementation of `AllocateLocallyUniqueId` which just increments a static atomic or something within the `dxvk` namespace, ifdef the entire function definition, and remove the `::` from the call.

Reply:
Ye, my initial impl was that. I will go back to it

Reply:
Done.